staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -193,10 +193,10 @@ func RecordLongRunning(req *http.Request, requestInfo *request.RequestInfo, fn f
 func MonitorRequest(req *http.Request, verb, resource, subresource, scope, contentType string, httpCode, respSize int, elapsed time.Duration) {
 	reportedVerb := cleanVerb(verb, req)
 	client := cleanUserAgent(utilnet.GetHTTPClient(req))
-	elapsedMicroseconds := float64(elapsed / time.Microsecond)
+	elapsedSeconds := float64(elapsed / time.Second)
```

Comment:
Please test this until you find the bug.

Reply:
@mikkeloscar the underlying type is `int64`